cluster/ubuntu/util.sh

```diff
@@ -381,9 +385,13 @@ function provision-master() {
   fi
 
   # remote login to MASTER and configue k8s master
+  config_file_name=$(basename ${KUBE_CONFIG_FILE})
```

Comment:
etc. .. just sweep this file and assume I'm going to nitpick each one. :)

You might `apt-get install shellcheck` and run it over and look for anything obvious. It's pretty nitpicky about quotes, and I'll use it once I stop seeing the obvious ones. (I wish we had it as a precommit.)